File: csharp/DniRucClient.cs
// Consulta DNI y RUC — mblapirest.com
using System;
using System.Net.Http;
using System.Net.Http.Headers;
using System.Threading.Tasks;

class DniRucClient
{
    private const string Token = "<tu_token>";
    private const string Base = "https://mblapirest.com/api/v1";

    static async Task<string> Consultar(HttpClient cliente, string ruta)
    {
        var respuesta = await cliente.GetAsync($"{Base}{ruta}");
        return await respuesta.Content.ReadAsStringAsync();
    }

    static async Task Main()
    {
        using var cliente = new HttpClient();
        cliente.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", Token);

        Console.WriteLine(await Consultar(cliente, "/dni/12345678"));
        Console.WriteLine(await Consultar(cliente, "/ruc/20123456789"));
    }
}

File: swift/DniRucClient.swift
// Consulta DNI y RUC — mblapirest.com
import Foundation

let token = "<tu_token>"
let base = "https://mblapirest.com/api/v1"

func consultar(_ ruta: String) async throws -> String {
    var peticion = URLRequest(url: URL(string: base + ruta)!)
    peticion.setValue("Bearer \(token)", forHTTPHeaderField: "Authorization")

    let (datos, _) = try await URLSession.shared.data(for: peticion)
    return String(data: datos, encoding: .utf8) ?? ""
}

Task {
    print(try await consultar("/dni/12345678"))
    print(try await consultar("/ruc/20123456789"))
}
